### PR TITLE
rtmp-services: Add Lahzenegar.com streaming platform

### PR DESCRIPTION
**Pull request by FarzadKarkhani**
- add Lahzenegar primary rtmp server

### CODE REVIEW

**Comment by kkartaltepe** [comment]
Be sure to increase the version number in `package.json` and fix your commit message to match the Commit Guidelines https://github.com/jp9000/obs-studio/blob/master/CONTRIBUTING.rst#commit-guidlines.

see #1026 for a proper PR following the commit message requirements and incrementing the version in `package.json`

**Comment by FarzadKarkhani** [comment]
It's done.

**Comment by jp9000** [comment]
You need to update this pull request with the package.json change -- not make a separate pull request for it.

**Comment by FarzadKarkhani** [comment]
Current pull request updated.

File: plugins/rtmp-services/data/services.json [hunk 1]
```diff
@@ -1104,6 +1104,19 @@
                 "max video bitrate": 20000,
                 "max audio bitrate": 192
             }
+        },
+        {
+            "name": "Lahzenegar - لحظه نگار",
```
Comment (harm27) [inline plugins/rtmp-services/data/services.json]: It is missing the following line: "common": false,

Comment (jp9000) [reply plugins/rtmp-services/data/services.json]: It seems you did not see my previous message on the other pull request:  the value is false by default.  You do not need to use it unless you're setting it to true.

Comment (harm27) [reply plugins/rtmp-services/data/services.json]: Jep, missed that one. I have seen some use it earlier, so expected that these also needed it. Check line 1078 for that.

**Comment by FarzadKarkhani** [comment]
@RytoEX Thank you for reviewing my PR, It seems everything is fixed.

**Comment by jp9000** [comment]
Fixing the conflict and then merging.  You'll see "closed in [commit hash of the merged commit]".

**Comment by notr1ch** [comment]
@FarzadKarkhani  The hostnames for this service are no longer resolving. Has this service ceased operation?

**Comment by FarzadKarkhani** [comment]
> @FarzadKarkhani The hostnames for this service are no longer resolving. Has this service ceased operation?

Yes, it changes to rtmp://rtmp.lahzecdn.com:443/pro
I'll update it soon.

**Pull request closed without merging**